Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1365,14 +1365,16 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
 @inlinable
 @differentiable
 public func gelu<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
-  let ratio1 = Tensor<T>(0.7978845608, on: x.device)  // An approximation of √(2/π).
+  let xWithoutDerivative = withoutDerivative(at: x) { $0 }
```

Comment:
If `withoutDerivative(at: x)` is necessary, you can simplify it to this version:
```suggestion
  let xWithoutDerivative = withoutDerivative(at: x)
```

Reply:
Thanks. I did not realise I could write it that way.  I will be able to use `x` directly once #963 is merged.

Reply:
done now.